lib/addons/amp/index.ts

```diff
@@ -0,0 +1,144 @@
+import { KubernetesManifest } from "aws-cdk-lib/aws-eks";
+import * as aps from 'aws-cdk-lib/aws-aps';
+import { CfnWorkspaceProps } from 'aws-cdk-lib/aws-aps';
+import { ClusterAddOn, ClusterInfo } from "../../spi";
+import { dependable, loadYaml, readYamlDocument } from "../../utils";
+import { AdotCollectorAddOn } from "../adot";
+import { Construct } from 'constructs';
+import { CfnTag } from "aws-cdk-lib/core";
+
+/**
+ * Configuration options for add-on.
+ */
+ export interface AmpAddOnProps {
+    /**
+     * Name that will be used by the Addon to create the Workspace
+     * @default blueprints-amp-workspace
+     */
+    workspaceName?: string;
+    /** 
+     * Remote Write URL of the AMP Workspace to be used for setting up remote write.
+     */
+     prometheusRemoteWriteURL?: string;
+    /**
+     * Tags to passed while creating AMP workspace
+     * @default Project
+     */
+     workspaceTag?: CfnTag[];
```

Comment:
let's change it to workspaceTags, plural.

Reply:
Fixed.